docs/libcurl/opts/CURLINFO_CAINFO.md

```diff
@@ -10,6 +10,8 @@ See-also:
   - curl_easy_setopt (3)
 Protocol:
   - TLS
+TLS-backend:
+  - *
```

Comment:
is it your intention for it to render as `*` because I think `All` would be easier to understand

Reply:
I primarily think of machine parsing this, but using `All` is indeed a good idea for the human readers.